resources/wikia/modules/flowTracking.js

```diff
@@ -46,6 +38,17 @@ define('wikia.flowTracking', ['wikia.tracker', 'wikia.window', 'mw', 'jquery'],
 		return mw.config.get('wgNamespaceNumber') === 0;
 	}
 
+	function prepareParams(action, flow, extraParams) {
+		var params = {
+			action: action,
+			flowname: flow,
+			label: flow
+		};
+
+		extraParams = extraParams || {};
+		return $.extend(params, extraParams, { user_agent: userAgent });
```

Comment:
Not sure about the conventions we used here (or lack thereof). 
We have flowname, user_agent, eventName - can those be unified?